.github/workflows/ci-deep.yml

```diff
@@ -0,0 +1,149 @@
+name: "CI Deep"
+
+env:
+  FOUNDRY_PROFILE: "ci"
+
+on:
+  schedule:
+    - cron: "0 3 * * 0" # at 3:00am UTC every Sunday
+  workflow_dispatch:
+    inputs:
+      fuzzRuns:
+        default: "100000"
+        description: "Unit: number of fuzz runs."
+        required: false
+      invariantRuns:
+        default: "64"
```

Comment:
why did you set to 64 and 2048? 
I think I would prefer more runs than depth, but if you have goo reasons let me know

Reply:
It was a pseudo random value so let me know what values would you prefer

Reply:
Ok good then let's go with 300 runs and 50 depth